Frigate Camera Notifications/Stable

```diff
@@ -193,6 +193,19 @@ blueprint:
             - default
             - none
           custom_value: true
+    volume:
+      name: Volume Sound - iOS only (Optional)
+      description: You can spécify a sound level between 0 and 100
+      default: "100"
+      selector:
+        select:
```

Comment:
Perhaps this could be a slider? 0 to 100

Reply:
Absolutely, very good idea.